### PR TITLE
feat(charts) add experiment openebs-single-replica-failure

### PR DESCRIPTION
**Pull request by ispeakc0de**
- Add experiment openebs-single-replica-failure

**This PR contains**
- [x]  experiment.yaml 
- [x]  chartserviceversion.yaml

**Logs of the executor and job**
[logs-job.txt](https://github.com/litmuschaos/chaos-charts/files/3712095/logs-job.txt)
[logs-executor.txt](https://github.com/litmuschaos/chaos-charts/files/3712096/logs-executor.txt)

### CODE REVIEW

**Review by chandankumar4 — CHANGES_REQUESTED** [review]
Can you add the required permission also for running this experiment

**Comment by ksatchit** [comment]
@shubhamchaudhary , is the jiva replica failure a separate experiment OR can it be called as a lib from the pool chart?

**Comment by ispeakc0de** [comment]
> @shubhamchaudhary , is the jiva replica failure a separate experiment OR can it be called as a lib from the pool chart?

@ksatchit ,
We can modify our openebs-single-replica-failure experiment so that it will work for both cstor and jiva sc. We can Add lib for both jiva and cstor single replica failure.

File: charts/openebs/openebs-single-replica-failure/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description:
+  message: |
+    Delete the Jiva replica pod and ensure that it gets scheduled again
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: openebs
+  name: openebs-single-replica-failure
+  version: 0.1.0
+spec:
```
Comment (chandankumar4) [inline charts/openebs/openebs-single-replica-failure/experiment.yaml:12]: Can we have the permission field also

**Comment by ksatchit** [comment]
Do we have an experiment in this path Or is the plan to update the pool chaos tests to  work with jiva? @shubhamchaudhary

**Pull request closed without merging**